### PR TITLE
Add lj14scale and coulomb14scale to Forcefield

### LINKED ISSUE #397: Add LJ14Scale and coulomb14Scale as forcefield properties.
We can get them via following code:

```py
>>> non_bondedgen = oplsaa.get_generator(oplsaa, gen_type=NonbondedGenerator)
>>> non_bondedgen.lj14scale
0.5
>>> non_bondedgen.coulomb14scale
0.5
>>> 
```

See https://github.com/openforcefield/openff-system/pull/114#issuecomment-813573601

Thanks @daico007 for pointing out the correct way to get those values. All we need is a `property` descriptor at the forcefield level.

### PR DESCRIPTION
**Pull request by umesh-timalsina**
Closes #397.

### CODE REVIEW

**Review by daico007 — APPROVED** [review]
LGTM!

File: foyer/forcefield.py [hunk 1]
```diff
@@ -468,6 +468,30 @@ def included_forcefields(self):
             self._included_forcefields[basename] = ff_filepath
         return self._included_forcefields
 
+    @property
+    def lj14scale(self):
+        """Get LJ-14Scale for this forcefield"""
```
Comment (daico007) [inline foyer/forcefield.py:473]:
```suggestion
        """Get LJ 1-4 scale for this forcefield"""
```

File: foyer/forcefield.py [hunk 2]
```diff
@@ -468,6 +468,30 @@ def included_forcefields(self):
...
+        except MissingForceError:
+            raise AttributeError(
+                "Cannot get lj14Scale for the forcefield "
+                "because it doesn't have NonBondedForce."
+            )
+        return non_bonded_force_gen.lj14scale
+
+    @property
+    def coulomb14scale(self):
+        """Get coulomb14scale for this forcefield"""
```
Comment (daico007) [inline foyer/forcefield.py:485]:
```suggestion
        """Get Coulombic 1-4 scale for this forcefield"""
```

**Review by justinGilmer — APPROVED** [review]
LGTM, behavior is what I expect.

**Pull request merged**